Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -252,6 +252,16 @@ public extension CurrentChatUserController {
             }
         }
     }
+    
+    /// Get all blocked users.
+    ///
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func getBlockedUsers(completion: ((Error?) -> Void)? = nil) {
```

Comment:
We need to return the users, otherwise, the developer can't do nothing with this 😛

Reply:
Updated 👍